index.html

```diff
@@ -8,41 +8,80 @@
     <link rel="stylesheet" href="styles.css">
     <title>Portfolio Project</title>
   </head>
-  <body>
+  <body> 
+    <nav class="nav-bar">
+      <ul class="title-bar">
+        <li><a href="#" alt="">Hello</a></li>
+        <li><a href="#" alt="">Portfolio</a></li>
+        <li><a href="#" alt="">About</a></li>
+        <li><a href="#" alt="">Contact</a></li>
+      </ul>
+    </nav>
     <header>
       <nav class="main-menu">
         <a href="#"><img src="Images SVG/ic_menu.svg" alt=""></a>
        </nav>
     </header>
       
     <section class="headline">
-      <h1 class="title">Juan<br>Díaz</h1>
-      <p class="myname-intro">Hello! I am a software developer! I can
+      <div class="main">
+        <h1 class="title">Juan<br>Díaz</h1>
+        <p class="myname-intro">Hello! I am a software developer! I can
                               help you build a product, feature or
                               website. Take a look at my work.
                               If you like what you see and have a
                               project you need coded, don’t hesitate to
                               contact me.</p>
-      <button type="button" class="button-box">Start collaboration</button>
+        <button type="button" class="button-box">Start collaboration</button>
+        <div class="dot-grid"><img src="Images SVG/image_dots.svg" alt=""></div>
+      </div>
+      <div class="image-projects">
+        <div id="main-background1"><img src="Images SVG/image_geometry_2.svg" alt=""></div>
+        <div id="main-background2"><img src="Images SVG/image-geometry_1.svg" alt=""></div>
+      </div>
     </section>
```

Comment:
In the actual design the headline section appears to cover almost 50% with the image covering the remaining 50%. However, the image seems to cover most part of the width pushing the headline text to the far left as shown below :point_down: 
Kindly ensure that your design matches the actual design in terms of position, dimensions etc to make your project more professional. :pray: 

### Your design 

![Screenshot from 2023-04-08 00-56-21](https://user-images.githubusercontent.com/65233860/230696582-d1159a2b-11e0-4111-8854-d23fe1b20ce0.png)


### Actual design 

![Screenshot from 2023-04-08 00-56-33](https://user-images.githubusercontent.com/65233860/230696591-35a4f4f5-561b-499c-95cc-9020295cf7c9.png)